components/Header.tsx

```diff
@@ -1,9 +1,25 @@
 /* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
 
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Link from "next/link";
 import Transition from "./Transition";
 
+/**
+ * Indicates if the "Deploy" link in the header
+ * should be displayed to the user.
+ */
+function displayDeploy() {
+  const showDeploy = localStorage.getItem("showDeploy");
+  if (showDeploy !== null) {
+    return showDeploy === "true";
+  } else {
+    // 1% probability
+    const showDeploy = Math.random() < 1 / 100;
+    localStorage.setItem("showDeploy", String(showDeploy));
```

Comment:
it seems like over time, if people keep visiting the site they will eventually be pushed into the "showDeploy" category?

Reply:
The else block should only run on the first visit unless the "showDeploy" key is cleared from localStorage.